docker-compose.yaml

```diff
@@ -484,6 +484,64 @@ services:
       permissionless-builder:
         condition: service_healthy
 
+  marketplace-reserve-builder:
+    image: ghcr.io/espressosystems/espresso-sequencer/marketplace-builder:main
+    ports:
+      - "$ESPRESSO_BUILDER_SERVER_PORT:$ESPRESSO_BUILDER_SERVER_PORT"
+    profiles: [marketplace]
+    environment:
+      - ESPRESSO_MARKETPLACE_BUILDER_IS_RESERVE=true
+      - ESPRESSO_MARKETPLACE_BUILDER_SOLVER_URL=http://localhost:$ESPRESSO_MARKETPLACE_SOLVER_API_PORT
+      - ESPRESSO_MARKETPLACE_BUILDER_NAMESPACE=10000
+      - ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_URL=http://sequencer0:$ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_PORT
+      - ESPRESSO_SEQUENCER_STATE_PEERS=http://sequencer0:$ESPRESSO_SEQUENCER_API_PORT
+      - ESPRESSO_BUILDER_GENESIS_FILE
+      - ESPRESSO_BUILDER_ETH_MNEMONIC
+      - ESPRESSO_BUILDER_ETH_ACCOUNT_INDEX
+      - ESPRESSO_BUILDER_L1_PROVIDER
+      - ESPRESSO_BUILDER_SERVER_PORT
+      - ESPRESSO_BUILDER_TX_CHANNEL_CAPACITY
+      - ESPRESSO_BUILDER_EVENT_CHANNEL_CAPACITY
+      - ESPRESSO_BUILDER_BOOTSTRAPPED_VIEW
+      - ESPRESSO_BUILDER_WEBSERVER_RESPONSE_TIMEOUT_DURATION
+      - ESPRESSO_BUILDER_BUFFER_VIEW_NUM_COUNT
+      - ESPRESSO_BUILDER_INIT_NODE_COUNT
+      - RUST_LOG
+      - RUST_LOG_FORMAT
+      - ASYNC_STD_THREAD_COUNT
+    depends_on:
```

Comment:
We are using sequencer0 event streaming URL so this should also depend on sequencer0

Reply:
It already indirectly depends on sequencer through fund-builder, but explicitly stating it is probably safer